src/data/emission/ailleurs-312/description.html

```diff
@@ -0,0 +1,6 @@
+<p>
+Enregistré et diffusé sur <a href="https://radio.incongru.org/public/avenir">Radio Avenir</a> le 25 décembre après-midi<br/>
```

Comment:
```suggestion
Enregistré et diffusé sur <a href="https://radio.incongru.org/public/avenir">Avenir Radio</a> le 25 décembre après-midi<br/>
```

Sinon le jeu de mot ne fonctionne pas ^^